src/zinoargus/__init__.py

```diff
@@ -242,6 +242,7 @@ def synchronize_continuously(argus_incidents: IncidentMap, zino_cases: CaseMap):
         update = _notifier.poll(timeout=POLL_TIMEOUT)
         if not update:
             # No notification received (i.e. timeout occurred)
+            refresh_argus_incidents(argus_incidents, zino_cases)
```

Comment:
Having this here means it will only be updated if we don't have any new notifications from Zino - what if we have a period where there is a new case more than every 30 seconds? Then it will never be synced back to Zino

Reply:
We can at least sync back all the other incidents that are not currently updated